### PR TITLE
Add etc_hosts option for Molecule docker instances

### PR DESCRIPTION
**Pull request by tace**
Molecule tested docker instances cannot utilize
etc_hosts option of Ansible's docker_container
module as it's not implemented by Molecule's
docker driver's create.yml playbook.

This change implements option for Molecule user
to provide etc_hosts configuration in molecule.yml
file's platforms section.

Signed-off-by: Samuli Silvius <samuli.silvius@gmail.com>

Please include details of what it is, how to use it, how it's been tested

#### PR Type

- Feature Pull Request

### CODE REVIEW

File: molecule/driver/docker.py [hunk 1]
```diff
@@ -114,7 +117,7 @@ class Docker(base.Base):
         - name: instance
           image: centos:7
           privileged: true
-          volume_mounts:
+          volumes:
```
Comment (decentral1se) [inline molecule/driver/docker.py]: Can we put this unrelated change into another commit?

Comment (tace) [reply molecule/driver/docker.py]: yes please

**Comment by fabiencytadel** [comment]
FYI ; works as expected, tested in our lab

**Comment by ssbarnea** [comment]
Please fix conflicts on this and also assure that it does not contian unrelated changes. Once done I will merge it.

**Comment by ssbarnea** [comment]
No reply, closing.
